README.md

```diff
@@ -1,5 +1,54 @@
-# 2dTo3dVector
-## Additions
-A CLI to convert 2D vector data to 3D vector data. 
-Testing a commit with VSCode
-Now, submitting a Pull Request instead of a Commit
\ No newline at end of file
+# Mountaineer
+A CLI to convert 2D vector data to 3D vector data.  
+
+## Usage
+CLI Usage: One liner
+
+mountaineer [input file] [lat] [lon] [output file]
+
+[input file]: Input file. File type can be .KML, .SHP, .GeoJSON (preferred). These are geospatial formats that contain Lat/long coordinates. Input file length should not exceed 128 characters. 
+
+[lat]: Latitude field. Required to create WKT
+[lon]: Longitude field. Required to create WKT
+
+[output]: User-input Output file name. File will be placed in the same folder as the input file. File length should not exceed 128 characters. File output will DEFAULT to .GeoJSON, but other outputs can be selected by changing the file extension. 
+
+## Other Commands
+Other commands:
+
+mountaineer – help
+	Brings up help: shows a baseline command for users, accepted file types, accepted inputs and outputs
+
+mountaineer q - exits the CLI and returns user to terminal 
+
+
+### Description of Process
+
+Locally
+-CLI awaits for user input, user input being a path to a 2d geometry file stored locally.
+-File can be points, lines, or polygon for geometry
+-File is opened in Geopandas within a new dataframe
+-File is checked for proper geometry values for its data type, type of file is read back via console for verification.
+-File is scanned for latitude and longitude headers
+-Latitude and Longitude values are exported from file into a temporary file, as Well Known Text (WKT)
+
+AWS
```

Comment:
This process needs some tuning. But it is a good start.

Reply:
Still a work in progress, will need to find a way to make this work before. Will be updated as the structure is nailed down.